doc/en/deprecations.rst

```diff
@@ -37,11 +37,15 @@ display captured output when tests fail: ``no``, ``stdout``, ``stderr``, ``log``
 Node Construction changed to ``Node.from_parent``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. deprecated:: 5.3
+.. deprecated:: 5.4
 
 The construction of nodes new should use the named constructor ``from_parent``.
 This limitation in api surface intends to enable better/simpler refactoring of the collection tree.
 
+This means that instead of :code:``MyItem(name="foo", parent=collector, obj=42)``
+one now has to invoke :code:``MyItem.from_parent(collector, name="foo")``.
+
+
```

Comment:
Could also have an example (or mention) that you have to use keyword arguments now, i.e. with mypy it has to look like this:
```diff
@ mypy/test/data.py:538 @ def split_test_cases(parent: 'DataSuiteCollector', suite: 'DataSuite',
     for i in range(1, len(cases), 6):
         name, writescache, only_when, platform_flag, skip, data = cases[i:i + 6]
         platform = platform_flag[1:] if platform_flag else None
-        yield DataDrivenTestCase(parent, suite, file,
+        yield DataDrivenTestCase.from_parent(parent, suite=suite, file=file,
                                  name=add_test_name_suffix(name, suite.test_name_suffix),
                                  writescache=bool(writescache),
                                  only_when=only_when,
```

Reply:
done, thanks